[MPS] Enable log1p and sigmoid for int64

malfet opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #151791
* #151790

It works on MacOS-15, but likely will need a skip for MacOS-13

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

malfet committed:
[MPS] Enable log1p and sigmoid for int64

It works on MacOS-15, but likely will need a skip for MacOS-13

[ghstack-poisoned]

kulinseth reviewed (commented):
Yes I think this was fixed in 14 onwards releases

malfet commented:
> Yes I think this was fixed in 14 onwards releases

But if https://github.com/pytorch/pytorch/actions/runs/14575177882/job/40880028627 to be believed, looks like it just works

malfet committed:
Update on "[MPS] Enable log1p and sigmoid for int64"


It works on MacOS-15, but likely will need a skip for MacOS-13

[ghstack-poisoned]

malfet committed:
Update on "[MPS] Enable log1p and sigmoid for int64"


It works on MacOS-15, but likely will need a skip for MacOS-13

[ghstack-poisoned]

malfet commented:
@pytorchbot merge -f "Lint + MPS tests are green"

qqaatw commented:
It passed on Ventura as well.
